.github/ISSUE_TEMPLATE/issue-default.yml

```diff
@@ -6,29 +6,27 @@ body:
     id: help
     attributes:
       value: |
-        > **Note**
-        > GitHub Issues use [GitHub Flavored Markdown](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax) for formatting.
+        > Titles should be short, descriptive, and compelling.
   - type: textarea
     id: issue
     attributes:
-      label: Issue Description
+      label: Issue description and context
       description: |
-        Describe the issue you are adding or content you are suggesting.
-        Share any next steps that should be taken our outcomes that would be beneficial.
+        Describe the issue so that someone who wasn't present for its discovery can understand the problem and why it matters. Use full sentences, plain language, and good [formatting](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax – links and images welcome! Share desired outcomes or potential next steps.
     validations:
       required: true
   - type: textarea
-    id: additional-context
+    id: acceptance-criteria
     attributes:
-      label: Additional Context (optional)
-      description: "Include additional references (screenshots, design links, documentation, etc.) that are relevant"
+      label: Acceptance criteria
+      description: "If known, share 1-3 statements that would need to be true for this issue to be considered resolved. Use a checklist."
```

Comment:
if we are going to ask them to supply a checklist we could include a checklist in the default text of the block

Reply:
"If known" makes this an optional field, and it's going to be refined into a checklist either way by leads. My goal for this comment is to encourage the use of `- [ ]`, not to make them mandatory.

Reply:
>  I will say I disagree with using checkboxes, if the hope is for people to actually mark them off to show completion.

This is not my intent. It's really for product to glean a better understanding of the issue submitter's perspective of what done looks like – which, to Paul's point, we can then refine as needed. It's not (directly, immediately) for PR reviewers.